engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/jni/JniUtils.java

```diff
@@ -1680,6 +1680,15 @@ public static ByteBuffer getByteBuffer(PtNDArray ndArray) {
                 .order(ByteOrder.nativeOrder());
     }
 
+    public static ByteBuffer getDirectByteBuffer(PtNDArray ndArray) {
```

Comment:
replace `getByteBuffer()` with your implementation

Reply:
I still think we should return DirectBuffer in `getByteBuffer()`, I agree it's a risk that potential share the native memory between two tensors. But we can mitigate the risk by override`PtNDArray.copyTo()` function, and copy the buffer if it's a DirectBuffer